src/main/java/co/novu/api/tenants/TenantsApi.java

```diff
@@ -0,0 +1,35 @@
+package co.novu.api.tenants;
+
+import co.novu.api.tenants.requests.TenantRequest;
+import co.novu.api.tenants.responses.BulkTenantResponse;
+import co.novu.api.tenants.responses.DeleteTenantResponse;
+import co.novu.api.tenants.responses.TenantResponse;
+import retrofit2.Call;
+import retrofit2.http.Body;
+import retrofit2.http.DELETE;
+import retrofit2.http.GET;
+import retrofit2.http.PATCH;
+import retrofit2.http.POST;
+import retrofit2.http.Path;
+import retrofit2.http.Query;
+
+public interface TenantsApi {
+
+    String ENDPOINT = "tenants";
+
+    @GET(ENDPOINT)
+    Call<BulkTenantResponse> getTenants(@Query("page") Integer page, @Query("limit") Integer limit);
+
+    @POST(ENDPOINT)
+    Call<TenantResponse> createTenant(@Body TenantRequest request);
+
+    @GET(ENDPOINT + "/{tenantId}")
+    Call<TenantResponse> getTenant(@Path("tenantId") String tenantId);
+
+    @PATCH(ENDPOINT + "/{tenantId}")
+    Call<TenantResponse> updateTenant(@Body TenantRequest request, @Path("tenantId") String tenantId);
+
+    @DELETE(ENDPOINT+ "/{tenantId}")
+    Call<DeleteTenantResponse> deleteTenant(@Path("tenantId") String tenantId);
```

Comment:
This should return Call<Void> and not Call< DeleteTenantResponse>

Reply:
changed